omod/src/main/webapp/pages/myaccount/changeDefaults.gsp

```diff
@@ -1,5 +1,5 @@
 <%
-    ui.decorateWith("appui", "standardEmrPage", ["title": "Change User Defaults"])
+    ui.decorateWith("appui", "standardEmrPage", ["title": "My Languages"])
```

Comment:
Can translate the title please?

Reply:
@wluyima Thanks for reviewing my changes, but I haven't gotten what you meant by " Can translate the title please? "

Reply:
Internationalize it, don't hard code the string "My Language"